qubesappmenus/__init__.py

```diff
@@ -151,7 +151,8 @@ def write_desktop_file(vm, source, destination_path, dispvm=False):
                 raise DispvmNotSupportedError()
             source = source. \
                 replace('\nExec=', '\nX-Qubes-NonDispvmExec='). \
-                replace('\nX-Qubes-DispvmExec=', '\nExec=')
+                replace('\nX-Qubes-DispvmExec=', '\nExec='). \
+                replace('\nName=', '\nName= (dvm) ')
```

Comment:
This will look rather bad in normal menu. Maybe convert to something like `Name=VMNAME (dvm): APPLICATION`? It will require a bit more advanced processing than just `replace` function, probably easiest done with regex.